portfolio2.css

```diff
@@ -0,0 +1,227 @@
+* {
+    box-sizing: border-box;
+	margin: 0;
+	padding:0;
+	font-family: ubuntu, sans-serif;
+}
+
+body {
+    margin: 0px;
+    /*background-image: url("https://images.pexels.com/photos/5044497/pexels-photo-5044497.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260");
+    background-color: #cccccc;*/
+
+   }
+	
+  div.links {
+		border-bottom-style: solid;
+		border-bottom-color: Lightgrey;
+		list-style: none;
+		margin: 0 2px;
```

Comment:
2px side margin in combination with width: 100% is causing a sidescrolling bar to appear. Would remove, along with width: 100%